downstream/modules/platform/con-gw-understanding-authenticator-mapping.adoc

```diff
@@ -0,0 +1,95 @@
+
+For example, let's simplify the in-memory representation of the default permissions as follows:
+
+-----
+Access allowed = True
+Superuser permission = Undefined
+Admin of teams = None
+-----
+And, let’s say the maps which need to be processed are processed in the following order:
+
+. *Allow* rule set to never
+. *Allow* rule based on group
+. *Superuser* rule based on user attributes
+. *Team* admin rule based on user group
+
+The first *Allow* map, set to never, denies access to the system and the in memory representation would now look like:
+
+[subs=+quotes]
+-----
+Access allowed = *False*
+Superuser permission = Undefined
+Admin of teams = None
+-----
+
+However, if the user matches the second *Allow* map (the group-based allow), the permissions change to the following:
+
+[subs=+quotes]
+-----
+Access allowed = *True*
+Superuser permission = Undefined
+Admin of teams = None
+-----
+
+Where the user is subsequently granted access to Ansible Automation Platform because they have the required groups. 
+
+Next, the *Superuser* map checks user attributes. If no match is found, it does not revoke existing permissions by default. Therefore, the permissions remain the same as the results from our last map:
+
+[subs=+quotes]
+-----
+Access allowed = True
+Superuser permission = *Skipped*
+Admin of teams = None
+-----
+
+To revoke superuser access, you can select the *Revoke* option on the *Superuser* map. That way, when the user does not meet the attribute criteria, the permissions update to False like in the following:
+
+[subs=+quotes]
+-----
+Access allowed = True
+Superuser permission = *False*
+Admin of teams = None
+-----
+
+The final *Team* map checks the user’s groups coming from the authenticator for admin access on the team “My Team”. If the user has the required group, the permissions update to the following:
+
+[subs=+quotes]
+-----
+Access allowed = True
+Superuser permission = False
+Admin of teams = “*My Team*”
+-----
+
+If the user lacks the required group, permissions remain unchanged unless the *Revoke* option has been selected on the map, in which case permissions update to the following:
+
+[subs=+quotes]
+-----
+Access allowed = True
+Superuser permission = False
+Admin of teams = *Revoke admin of “My Team”*
+-----
+After processing all maps in the order defined, the final permissions reconcile, updating the user’s access based on the map rules.
+
+In summary, authenticators authenticate users and delegate system authorization to the authenticator maps defined for the authenticator. Authenticator maps are executed in order creating an in memory representation of the users permissions which get reconciled with the users actual permissions once all maps are executed.
```

Comment:
Just thinking of ways to be less redundant with the use of "authenticate" and "authenticator"

```suggestion
In summary, authenticators validate users and delegate system authorization to the authenticator maps. Authenticator maps are executed in order creating an in-memory representation of the users permissions which get reconciled with the users actual permissions after all maps are executed.
```

Reply:
Fixed.